.github/workflows/conformance-multicluster.yaml

```diff
@@ -277,7 +277,7 @@ jobs:
         run: |
           cilium hubble port-forward --context ${{ steps.contexts.outputs.context1 }}&
           sleep 10s
-          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 2 ]]
+          [[ $(pgrep -f "cilium hubble port-forward|kubectl port-forward.*hubble-relay" | wc -l) == 4 ]]
```

Comment:
> `cilium hubble port-forward` is run for both clusters, so we expect to see each process 2 times.

Wait, am I missing something? We don't run `cilium hubble port-forward` for the second cluster, and it doesn't even have Relay 🤔

Reply:
You're completely right, sorry. I must have mixed something up. I'll drop the commit again. Though it's still a mystery to me why that check is consistently failing.